src/commands/build/watch/mod.rs

```diff
@@ -2,19 +2,18 @@ mod watcher;
 pub use watcher::wait_for_changes;
 
 use crate::commands::build::{command, wranglerjs};
-use crate::commands::publish::Package;
 use crate::settings::project::{Project, ProjectType};
 use crate::terminal::message;
 use crate::{commands, install};
 
 use notify::{self, RecursiveMode, Watcher};
-use std::env;
 use std::sync::mpsc;
 use std::thread;
 use std::time::Duration;
 
 pub const COOLDOWN_PERIOD: Duration = Duration::from_millis(2000);
 const JAVASCRIPT_PATH: &str = "./";
```

Comment:
For JavaScript I guess we are watching `node_modules` as well.

Reply:
Yup!